certified-connectors/Calendar Pro/Readme.md

```diff
@@ -0,0 +1,41 @@
+## TeamsPro - Calendar Pro Connector
```

Comment:
Readme file is not properly formatted. Please follow this [link](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) for template readme and add required sections.